conjure-java-core/build.gradle

```diff
@@ -38,13 +38,15 @@ dependencies {
     compile 'com.google.googlejavaformat:google-java-format'
     compile 'com.google.guava:guava'
     compile 'com.palantir.conjure.java.api:errors'
+    compile 'com.palantir.dialogue:dialogue-client'
     compile 'com.palantir.ri:resource-identifier'
     compile 'com.palantir.syntactic-paths:syntactic-paths'
     compile 'com.palantir.tokens:auth-tokens'
     compile 'com.squareup:javapoet'
     compile 'javax.annotation:javax.annotation-api'
     compile 'javax.validation:validation-api'
     compile 'org.apache.commons:commons-lang3'
+    compile 'org.glassfish.jersey.core:jersey-common' // TODO(rfink): only here for UriTemplateParser. Re-implement?
```

Comment:
would prefer to do this before merging

Reply:
will do

Reply:
Second thought: if I touch this then I would want to implement the "Unique Conjure API endpoint paths" proposal (will send internal link). That's a somewhat bigger project so I'd prefer to merge this as is and then re-visit the path/routing implementation more holistically later.